apps/rehabstod/src/components/Table/tableHeader/TableHeaderCell.tsx

```diff
@@ -27,9 +37,9 @@ export function TableHeaderCell({ column, width, sticky }: { column: string; wid
               ;(currentTarget.nextElementSibling as HTMLElement).focus()
             }
           }}
-          onClick={() => sortOnColumn(column)}
+          onClick={unSortable ? () => {} : () => sortOnColumn(column)}
```

Comment:
`sortOnColumn` gets called in `onKeyDown` as well.

Reply:
Since you can't tab to it I don't think this is possible to call but I can remove it

Reply:
Good point